src/main/java/com/healthapp/project/Entity/MedicationDex.java

```diff
@@ -0,0 +1,15 @@
+package com.healthapp.project.Entity;
+
+
+import io.norberg.automatter.AutoMatter;
+
+@AutoMatter
+public interface MedicationDex {
+
+    String brand_name();
+
+    String strength();
+
+    String dosage_form();
+
+}
```

Comment:
using my own object here for the medication db. i didn't include id here because i won't be getting that from the api. When the builder class for this object tries to build the object without the id, i get an error. Don't worry about this. This is just needed to fill the table outright.